root/var/lib/vz/snippets/exec-cmds

```diff
@@ -27,6 +27,22 @@ exec_pci_rescan() {
   echo 1 > /sys/bus/pci/rescan
 }
 
+exec_set_haltpoll() {
+  echo "Setting haltpoll for $VMID..."
+  echo $1 > /sys/module/kvm/parameters/halt_poll_ns
+}
+
+exec_assign_interrupts() {
+  echo "Wating 30 seconds for all vfio-gpu interrupts to show up..."
+  sleep 30s
+  echo "Moving $1 interrupts to $2 cpu cores $VMID..."
+  grep $1 /proc/interrupts | cut -d ":" -f 1 | while read -r i; do
+        echo $i
+        MASK=$2
+        echo $MASK > /proc/irq/$i/smp_affinity
```

Comment:
It would be better to assign to `smp_affinity_list` that accepts a human-readable form, like: `0-11`.

Reply:
Ok, I will test it though, I agree that it's better from a usability perspective.

Reply:
Changed the script to assign to smp_affinity_list. Works as expected.